recipes/urllib3/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.16" %}
+
+package:
+  name: urllib3
+  version: {{ version }}
+source:
+  fn: urllib3-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/u/urllib3/urllib3-{{ version }}.tar.gz
+  md5: fcaab1c5385c57deeb7053d3d7d81d59
```

Comment:
I've been sticking with MD5s because that seems to be the default created by `conda skeleton`. Switching to `sha256` is fine by me - we should also tweak the example recipe to use `sha256` as well.